src/main/java/seedu/address/MainApp.java

```diff
@@ -81,6 +81,11 @@ public void init() throws Exception {
         modelManager.addEventListListener(uiManager);
 
         uiManager.addCommandInputListener(commandManager);
+
+        notificationCheckingThread = new NotificationCheckingThread(modelManager);
+        notificationCheckingThread.addPopupListener(new PopupListener(new SystemTrayCommunicator()));
+        notificationCheckingThread.setDaemon(true);
+        notificationCheckingThread.start();
```

Comment:
Good job, everything is working okay. It would be good to move these lines in the future to a `NotificationManager.start()`, to adhere to SLAP.

Reply:
Ok, will do.